lxd/storage/backend_lxd.go

```diff
@@ -3621,7 +3621,10 @@ func (b *lxdBackend) CreateInstanceSnapshot(inst instance.Instance, src instance
 		defer func() { _ = src.Unfreeze() }()
 
 		// Attempt to sync the filesystem.
-		_ = filesystem.SyncFS(src.RootfsPath())
+		err = filesystem.SyncFS(src.RootfsPath())
+		if err != nil {
+			l.Warn("Failed to flush writes to instance volume", logger.Ctx{"err": err})
```

Comment:
Ah good, you did sync the comment here, thanks (see my earlier comment).

However there are more things to consider along this theme.